atlp-rwanda/fan7-bn-frontend
# #170764520 integrate test coverage badge

### What does this PR do?
Add test coverage badge to the readme file
### Description of Task to be completed?
 - Integrate code climate test coverage into our project
 - Configure jest coverage reporters
 - Add test coverage badge to the readme file
### How should this be manually tested?
Visit this repo and read the readme file you will see a badge called `test coverage`
### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#170764520](https://www.pivotaltracker.com/story/show/170764520)
### screenshot
![coverage](https://user-images.githubusercontent.com/50277906/76604670-d5922780-6517-11ea-9ec6-40ac1adff550.PNG)
